packages/server/lib/cloud/api.ts

```diff
@@ -452,23 +465,40 @@ module.exports = {
     responseCache = {}
   },
 
-  preflight (preflightInfo) {
+  postPreflight (preflightInfo) {
```

Comment:
this was renamed to postPreflight but there are several places preflight is referenced in this file.

Reply:
Seems like this should still be named `preflight`

Reply:
where is it referenced? all of the other methods are verbs, so I just renamed it to make it consistent

Reply:
ohh ha, teaches me to read it a bit closer. I was reading it as `postFlight`